mllib/src/test/scala/org/apache/spark/ml/clustering/LDASuite.scala

```diff
@@ -119,6 +121,8 @@ class LDASuite extends SparkFunSuite with MLlibTestSparkContext with DefaultRead
     assert(lda.getLearningDecay === 0.53)
     lda.setLearningOffset(1027)
     assert(lda.getLearningOffset === 1027)
+    lda.setEpsilon(1e-3)
```

Comment:
here you should test a value different from the default, otherwise this tests nothing.

Reply:
Thanks @mgaido91 , I will update the code.

Reply:
Thanks.